content/body/video-content.php

```diff
@@ -50,19 +50,19 @@
 </p>
 
 <p>What makes a video accessible is widely misunderstood. Many web professionals know about closed captions.
-  What many don't know is that they absolutely need audio descriptions in order to be WCAG AA compliant.</p>
+  What many don't know is that they need audio descriptions to be WCAG AA compliant.</p>
 
   <?php include "includes/wcag-video-table.php"; ?>
 
 <p>
-  To find one of the most cost-effective way of implementing all three, I would suggest looking into the <a href="video-player.php">Enable Video Player</a> page, which shows how do so using <a href="https://ableplayer.github.io/ableplayer/">Able Player</a>.
+  To find one of the most cost-effective ways of implementing all three, I would suggest looking into the <a href="video-player.php">Enable Video Player</a> page, which shows how to do so using <a href="https://ableplayer.github.io/ableplayer/">Able Player</a>.
 </p>
 
 
 
 
 
-<h2>Should I Use Auto Generated Captions For My Videos?</h2>
+<h2>Should I use auto-generated captions For My Videos?</h2>
```

Comment:
Either capitalize everything in the header, or nothing. And it should be consistent throughout the page.

```suggestion
<h2>Should I Use Auto-Generated Captions For My Videos?</h2>
```